### DISCUSSION TITLE
如何使用glm-4的api服务

### DISCUSSION THREAD

**Discussion by RuipingWang1986** [modelscope/ms-agent#412] [Q&A]
我想要使用glm-4的api服务，但是配置总是有问题，请问有没有相关教程或者脚本提供？

**Comment by zzhangpurdue** [comment]
https://mp.weixin.qq.com/s/hj8jM-kgksuOWl4v86tRYQ
参考这篇文章，我们会把modelscope-agent相关社区中文文档，后面会补充起来到目录docs底下
